src/rules/no-same-line-conditional.ts

```diff
@@ -30,6 +30,7 @@ interface SiblingIfStatement {
 
 const rule: Rule.RuleModule = {
   meta: {
+    type: "problem",
```

Comment:
should be `suggestion`, as there is big chance it's just formatting problem (https://jira.sonarsource.com/browse/RSPEC-3972 is code smell)